Automattic/newspack-plugin
# feat(ads): Add-Ons and Ad Refresh Control integration

### All Submissions:






### Changes proposed in this Pull Request:

Implements Ads Add-Ons wizard and [Ad Refresh Control](https://wordpress.org/plugins/ad-refresh-control) plugin settings management.

<img width="1531" alt="image" src="https://user-images.githubusercontent.com/820752/159058166-c4e94f3f-5e6e-43c9-9f3b-f4e29425c8d0.png">

<img width="1528" alt="image" src="https://user-images.githubusercontent.com/820752/159058333-1b3ca071-4c5d-4db6-b6f8-41b7f79471f4.png">

Closes https://github.com/Automattic/newspack-ads/issues/299.
Closes https://github.com/Automattic/newspack-ads/issues/176.

### How to test the changes in this Pull Request:

1. Check out this branch, https://github.com/Automattic/newspack-ads/pull/346, and visit the new page at "Advertising -> Add-Ons"
2. If you have SCAIP already installed, toggle it off, visit the Settings wizards and confirm no SCAIP settings are displayed
3. Toggle the Ad Refresh Control and visit the Settings wizard
4. Modify Ad Refresh Control settings to custom values and confirm they are persisted by visiting the original plugin settings at **Settings -> Ad Refresh Control**
5. Confirm the settings are applied by inspecting the DOM and looking for `var AdRefreshControl`
6. With AMP Plus, confirm Ad Refresh Control frontend scripts are also preserved:

<img width="1068" alt="image" src="https://user-images.githubusercontent.com/820752/160181191-813495c5-9433-4820-b46a-5442bcfcbefe.png">

7. Toggle the the plugin settings off and confirm it no longer load the frontend scripts


### Other information:
